src/types.ts

```diff
@@ -8,6 +8,9 @@ import {Request, Response} from '@loopback/rest';
 import {VerifyFunction} from './strategies';
 export * from './strategies/types';
 
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+export type AnyObject = Record<string, any>; // NOSONAR
+
```

Comment:
instead of this use @loopback/repository

Reply:
no need to create your own type 
this will change the imports everywhere

Reply:
ok sure